Database/Corrections/cataQuestFixes.lua

```diff
@@ -546,6 +546,10 @@ function CataQuestFixes.Load()
         [9369] = { -- Replenishing the Healing Crystals
             [questKeys.requiredRaces] = raceKeys.ALL_ALLIANCE - raceKeys.DRAENEI,
         },
+        [9436] = { -- Bloodscalp Insight
+            [questKeys.preQuestSingle] = {},
+            [questKeys.preQuestGroup] = {26280,26279},
```

Comment:
Same here. How do you know both quests are required for the pick up?

Reply:
Easiest way to check for this is to turn in the quest that is labeled prequestsingle in quest `first`. If the followup doesn't pop, then it's a prequestgroup

Reply:
I tested this with two characters, turning in a different quest first each time - the follow up didn't appear until both were handed in. I will try a third character tomorrow just to be sure.

Reply:
Retested with another two characters - the prequests are actually The Defense of Grom'gol: Murloc Menace (26280) and I Think She's Hungry (26321)